xbout/plotting/utils.py

```diff
@@ -153,8 +153,8 @@ def plot_separatrices(da, ax):
 
     j11, j12, j21, j22, ix1, ix2, nin, nx, ny, y_boundary_guards = _get_seps(da)
 
-    R = da.coords['R'].transpose('x', 'theta')
-    Z = da.coords['Z'].transpose('x', 'theta')
+    R = da.coords['R'].transpose('x', 'theta').values
```

Comment:
Something to bear in mind is that this approach isn't robust to changing the name of `x`. 

If you use `open_stormdataset`, which changes `x` -> `radial`, then this would fail
```python
ds = open_stormdataset(files)
ds.bout.plot_seps() # (or whatever the syntax is now)
```